src/pages/boards/create.tsx

```diff
@@ -0,0 +1,9 @@
+import CreateArticle from '@/src/containers/boards/CreateArticle/CreateArticle'
+
+export default function createArticlePage() {
+  return (
+    <>
+      <CreateArticle />
+    </>
```

Comment:
```suggestion
    <CreateArticle />
```
불필요한 fragment인것같아요!

혹시 CreateArticle컴포넌트를 따로 빼신 이유가 있으실까요??